crates/committed/src/report.rs

```diff
@@ -63,7 +63,7 @@ pub(crate) enum Content<'s> {
     NoPunctuation(NoPunctuation),
     Imperative(Imperative<'s>),
     Wip(Wip),
-    Fixup(Fixup),
+    Autosquash(Autosquash),
```

Comment:
When splitting out a PR, please address feedback from the old one as its easy for it to drop through the cracks otherwise, see https://github.com/crate-ci/committed/pull/394#discussion_r1761723636

The PR is marked as a breaking change with `!` but it doesn't say *how* its a breaking change and I don't think making a breaking change for this is worth it.

If you still want to move forward with this, you could leave the message as `fixup` and but have an alias for `autosquash` and an issue tracking the switch over / removing of aliases.

Reply:
> The PR is marked as a breaking change with ! but it doesn't say how its a breaking change

I believed that the title of the PR and commit was self explanatory. If there's anything unclear about it, please consider adjusting the commit message when or if you squash and rebase.

Reply:
I am asking because it is not clear. Even if it was, I would ask that you not out source fixing of your PRs to maintainers as that is not a scalable solution as there are significantly more contributors to maintainers.

Reply:
I have amended the commit message to make the breaking change clear; however, I am still unsure whether this is something you would like or reject, or would require amendments to.

You suggested inverting the logic to at an autosquash alias. Is this a requirement to get this merged? Personally, keeping the name `fixup` when it handles all of Git's autosquash commits would be confusing to maintain later on; but I understand that from a functional perspective, the alias works symmetrically.

Reply:
> I have amended the commit message to make the breaking change clear; 

The commit (but not PR) says

> BREAKING CHANGE: rename fixup to autosquash

That doesn't say in what contexts.  In *most* cases, there is no breaking change for this iiuc. this is specifically scoped to the json messages that are emitted.

> You suggested inverting the logic to at an autosquash alias. Is this a requirement to get this merged? 

My suggestion is specifically for the json messages we emit.  By removing that breaking change (and ideally having no other), it changes this PR from a "unsure" to "yeah, lets merge this".

As I said, we can track re-invert of the json message in an issue along with removing the references to fixup.  This might not be there always but it will likely be there for a while.